{{cookiecutter.app_name}}/config/app.toml

```diff
@@ -0,0 +1,5 @@
+[debug]
```

Comment:
What about adding shared params values for HTTP address and port like in Beego?
![image](https://user-images.githubusercontent.com/696529/108025081-66428300-7058-11eb-8f0e-4ed9c659fa18.png)

Reply:
Sounds good. Fixed in 190a010690dd634483660246d2f160b13ef11f67

You can get the config value using helpers function for configs that depend on `gin.Mode()` and can use viper directly for config that not depend on `gin.Mode()`.